src/test/java/com/elasticbox/jenkins/tests/PullRequestManagerTest.java

```diff
@@ -0,0 +1,255 @@
+        final String triggerPhrase = TEST_TRIGGER_PHRASE;
+        pullRequest.comment(triggerPhrase + (count++) );
+        Assert.assertNull(MessageFormat.format("Unexpected build triggered with comment ''{0}''",
+                triggerPhrase),
+                waitForNextBuild(project, NEXTBUILD_TIMEOUT, "Waiting build triggered with comment " + triggerPhrase));
+
+        updateTriggerPhrase(project, triggerPhrase);
+        updateTriggerPhrase(project2, triggerPhrase);
+        pullRequest.comment(triggerPhrase + (count++) );
+        // Next build trigger is the same for both projects. Testing one of them is enough
+        waitForNextBuild(project, NEXTBUILD_TIMEOUT, "ensureBuildTriggeredInProject");
+
+        // Test both projects building this pullRequest and this commit
+
+        LOGGER.fine("Check that the job 1 build is started");
+        waitForBuildTriggered(project, NEXTBUILD_TIMEOUT_LONG, "testAddingProjectToGHRepository 1");
+
+        FreeStyleBuild projectLastBuild1 = project.getLastBuild();
+        if (projectLastBuild1 == null) {
+            LOGGER.fine("Build 1 is null");
+        } else {
+            LOGGER.fine("The job 1 have triggered build with number " + projectLastBuild1.number);
+        }
+
+        Assert.assertNotNull(MessageFormat.format("Build 1 is not triggered on opening of pull request {0} after {1} minutes",
+                pullRequest.getGHPullRequest().getHtmlUrl(), TimeUnit.SECONDS.toMinutes(NEXTBUILD_TIMEOUT_LONG)),
+                project.getLastBuild() );
+
+
+        LOGGER.fine("Check that the job 2 build is started");
+        waitForBuildTriggered(project2, NEXTBUILD_TIMEOUT, "testAddingProjectToGHRepository 2");
+
+        FreeStyleBuild projectLastBuild2 = project2.getLastBuild();
+        if (projectLastBuild2 == null) {
+            LOGGER.fine("Build 2 is null");
+        } else {
+            LOGGER.fine("The job 2 have triggered build with number " + projectLastBuild2.number);
+        }
+
+        Assert.assertNotNull(MessageFormat.format("Build 2 is not triggered on opening of pull request {0} after {1} minutes",
+                pullRequest.getGHPullRequest().getHtmlUrl(), TimeUnit.SECONDS.toMinutes(NEXTBUILD_TIMEOUT_LONG + NEXTBUILD_TIMEOUT)),
+                project2.getLastBuild());
+
+        // Tests success. Finishing...
+        LOGGER.fine("Test completed succesfully. Cleaning...");
+
+        // Ensure triggered builds are completed
+        waitForCompletion(project, TimeUnit.MINUTES.toSeconds(3) );
+        Assert.assertFalse(MessageFormat.format("Build of pull request {0} for project {1} is still not complete after 3 minutes",
+                pullRequest.getGHPullRequest().getHtmlUrl(), project.getName()),
+                project.getLastBuild().isBuilding() );
+
+        waitForCompletion(project2, TimeUnit.MINUTES.toSeconds(3) );
+        Assert.assertFalse(MessageFormat.format("Build of pull request {0} for project {1} is still not complete after 3 minutes",
```

Comment:
Hmmmm, I don't like that hardcoded `3` in several places. Use a constant instead (or the existing constant *2, for example)

Reply:
Please, also change the comment to use the constant, just in case it is changed to avoid confusion with the trace